assignment-2/src/view/DumbComponents/QuestionsList.js

```diff
@@ -0,0 +1,131 @@
+// DUMB Component
+import React from 'react';
+import '../../App.css';
+import {UncontrolledCollapse, CardBody, Card} from 'reactstrap';
+import MyNavbar from './MyNavbar';
+import {FaChevronDown, FaChevronUp} from "react-icons/fa";
+
+
+const QuestionsList = ({
+                           questions, newQuestion, onCreate, onChange, sOUsername, filterTitle,
+                           onChangeFilter, onSearchClick, filterText, onSearchChange,
+                           onViewAnswers, onLogout
+                       }) => {
+
+
+    return (
+        <div>
+
+            <MyNavbar searchBar={true} loggedInAsButton={true} logoutButton={true} sOUsername={sOUsername}
+                      filterTitle={filterTitle} onChangeFilter={onChangeFilter} onSearchClick={onSearchClick}
+                      filterText={filterText} onSearchChange={onSearchChange} onLogout={onLogout}/>
+
+            <br/>
+
+            <div className="container" style={{"padding-top": "4em"}}>
+
+                <div>
+                    <div className="d-flex justify-content-center">
+                        <div className="my-auto">
+
+                        </div>
+                    </div>
+
+                    <br/>
+
+                    <div>
+                        <div className="d-flex justify-content-center" style={{"padding-bottom": "20px"}}>
+                            <div className="my-auto">
+                                <button id="toggler" className="align-items-center btn askButtonClass">Ask a question
+                                </button>
+                            </div>
+                        </div>
+
+                        <UncontrolledCollapse toggler="#toggler">
```

Comment:
You used react controls from a third party library. I explicitly asked for you to not do it (check the last line of the readme: "Do not use a React control library, just a (S)CSS library to style your HTML").